python/HHDNN.py

```diff
@@ -159,29 +159,29 @@ def __init__(self, isZZAnalysis, *args, **kwargs):
             ROOT.gROOT.ProcessLine(".L {}/interface/HHDNNinterface.h".format(base))
             ROOT.gROOT.ProcessLine(".L {}/interface/lester_mt2_bisect.h".format(base))
 
-        if not os.getenv("_HHbbttDNNInput"):
-            os.environ["_HHbbttDNNInput"] = "HHbbttDNNInput"
+        feat_file = "{}/{}/src/cms_runII_dnn_models/models/arc_checks/zz_bbtt/2021-11-22-0/" \
```

Comment:
Hi Elena, this file seems to be very ZZ->bbtt oriented, is it also used for HH->bbtt?